File: StandardFizzBuzz/fizzbuzz.js
const fizzbuzz = (x) => {
  if (x % 3 === 0 && x % 5 === 0) return "FizzBuzz";
  else if (x % 3 === 0) return "Fizz";
  else if (x % 5 === 0) return "Buzz";
  else return x;
};

for (let i = 1; i <= 100; ++i) console.log(fizzbuzz(i));

File: StandardFizzBuzz/fizzbuzz.cs
using System;

class FizzBuzz {
    static void Main() {
        for (uint i = 1; i <= 100; ++i) {
            if (i % 3 == 0 && i % 5 == 0)
                Console.WriteLine("FizzBuzz");
            else if (i % 3 == 0)
                Console.WriteLine("Fizz");
            else if (i % 5 == 0)
                Console.WriteLine("Buzz");
            else 
                Console.WriteLine(i);
        }
    }
}